docs/tutorials/integrate-dotwallet.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 8
+---
+
+# Tutorial 8: Integrating dotwallet
+
+
+[DotWallet](https://www.dotwallet.com/en) is a lightweight wallet designed to help users easily and securely manage their digital assets. We will introduce how to integrate dotWallet.
+
+
+## Dotwallet Developer Platform
+
+dotwallet uses [oauth 2.0 protocol](https://oauth.net/2/) for login.
+
+1. Before using dotwallet, you need to register and create a dApp on dotwallet [Developer Platform](https://developers.dotwallet.com/en).
+
+![](../../static/img/dotwallet-create-dapp.png)
+
+2. After creating the dApp, you will receive an email containing `app_id` and `secret`.
+
+
+![](../../static/img/dotwallet-mail.png)
+
+3. Next, you need to set [redirect-uris](https://www.oauth.com/oauth2-servers/redirect-uris)
```

Comment:
Explain more on what this is for.

Reply:
the authorization
        server redirects the user-agent back to the client using the
        redirection URI provided earlier (in the request or during
        client registration).